README.md

```diff
@@ -0,0 +1,495 @@
+# sanctuary-def
+
+sanctuary-def facilitates the definition of curried JavaScript functions which
+are explicit about the number of arguments to which they may be applied and the
+types of those arguments.
+
+```javascript
+const $ = require('sanctuary-def');
+
+const def = $.create($.env);
```

Comment:
I'm happy with that compromise.  At least now I know where to go for more information.